aclk/aclk_stats.c

```diff
@@ -245,8 +253,57 @@ static void aclk_stats_query_time(struct aclk_metrics_per_sample *per_sample)
     rrdset_done(st);
 }
 
+#ifdef ENABLE_NEW_CLOUD_PROTOCOL
+const char *rx_handler_get_name(size_t i);
+static void aclk_stats_newproto_rx(uint32_t *aclk_proto_tx_msgs_sample)
+{
+    static RRDSET *st = NULL;
+
+    if (unlikely(!st)) {
+        st = rrdset_create_localhost(
+            "netdata", "aclk_protobuf_tx_types", NULL, "aclk", NULL, "Received new cloud architecture messages by their type.", "msg/s",
+            "netdata", "stats", 200010, localhost->rrd_update_every, RRDSET_TYPE_LINE);
```

Comment:
```suggestion
            "netdata", "stats", 200010, localhost->rrd_update_every, RRDSET_TYPE_STACKED);
```

Perhaps stacked? So we can see the total number of msg/s (if that makes sense).

Reply:
i was thinking about that actually not sure why i choose line in the end. I will change it to stacked